target/linux/ath79/image/mikrotik.mk

```diff
@@ -4,7 +4,8 @@ define Device/mikrotik_routerboard-2011uias-2hnd
   $(Device/mikrotik_nand)
   SOC := ar9344
   DEVICE_MODEL := RouterBOARD 2011UiAS-2HnD
-  DEVICE_PACKAGES += kmod-usb2 kmod-i2c-gpio kmod-sfp kmod-gpio-beeper
+  DEVICE_PACKAGES += kmod-usb2 kmod-i2c-gpio kmod-sfp kmod-gpio-beeper \
+	kmod-switch-ar8xxx
```

Comment:
`kmod-switch-ar8xxx` is an empty package on ath79 target. We don't need to add it.